src/main/java/com/fasterxml/jackson/datatype/joda/cfg/JacksonJodaDateFormat.java

```diff
@@ -59,6 +62,7 @@ public JacksonJodaDateFormat(JacksonJodaDateFormat base,
     {
         super(base, useTimestamp);
         _formatter = base._formatter;
+        _formatterWithOffsetParsed = _formatter.withOffsetParsed();
```

Comment:
This probably could just be `base._formatterWithOffsetParsed`?

Reply:
The reason for this vanilla copy & paste is just in case `_formatter` is changed to something else in the future, so that you don't have to also maintain the value at `_formatterWithOffsetParsed`

The idea is, whatever is assigned to `_formatter`, create a with offset parsed version of exactly that.

Reply:
Do let me know if the reason I stated justifies it, if you still don't agree I will change it to what you have suggested.

Reply:
I don't have strong feelings here -- seems like avoiding another operation would be nice, but then again this is not in the hot codepath (as (de)serializer that uses it is cached). So I'll trust your judgment with it.